reflex-dom-core/ChangeLog.md

```diff
@@ -1,5 +1,9 @@
 # Revision history for reflex-dom-core
 
+## Unreleased
+
+* Breaking change: Remove HasJSContext and MonadJS. This change also removes the `js` type parameter from `Prerender`. Change `Prerender js t m` to `Prerender t m`.
```

Comment:
It was never clear to me what the JS context was for, and it wasn't documented. Is there any actual lost functionality, or was it just not used in practice?

Reply:
It was useful initially, but jsaddle seems to have taken over this role, and we don't need it any more. We've still to got to test it all the way through to mobile builds to make sure I haven't inadvertently broken something.

Reply:
But, no, there should be no loss in functionality.